projects/1.21-fabric/assets/rune-crafting/runes/lang/zh_cn.json

```
@@ -0,0 +1,15 @@
+{
+  "gui.runes.rune_crafting":"制作符文",
+  "block.runes.crafting_altar":"符文制作祭坛",
+  "block.runes.crafting_altar.hint":"更为经济地制作符文。",
+  "item.runes.small_rune_pouch":"符文袋",
+  "item.runes.medium_rune_pouch":"幻翼符文袋",
+  "item.runes.large_rune_pouch":"神秘符文袋",
+  "item.runes.rune_pouch.hint":"当装备上时，提供施法所需符文。",
+  "item.runes.arcane_stone":"奥术符文",
+  "item.runes.fire_stone":"火焰符文",
+  "item.runes.frost_stone":"冰霜符文",
+  "item.runes.healing_stone":"治愈符文",
```

```suggestion
  "item.runes.healing_stone":"治疗符文",
```

根据后面圣骑士&牧师mod，这里应该用治疗比较好一些。